src/guide/functions.md

```diff
@@ -14,7 +14,17 @@ module Main
 let add = (x, y) => x + y
 ```
 
-A function can perform a series of actions. One thing to note about functions in Grain is that by default they return the result of the final expression in the function body, without needing an explicit `return` statement.
```

Comment:
I think the new version makes sense but is less beginner-friendly. I think it would be beneficial to start with the part `A function can perform a series of actions.` still, and `functions in Grain by default return the result of the final expression without needing an explicit return` I think needs to be included in some sense I don't think the statement of `and functions evaluate to an expression` carries the same meaning to a beginner especially if they do not understand implicit return.

Reply:
`functions in Grain by default return the result of the final expression` is kind of misleading since function bodies are actually always just one expression (but can be a block expression). I'll leave a note in the line below about using block expressions with functions along the lines of what you're suggesting